.github/CORE_SERVICES.md

```diff
@@ -0,0 +1,27 @@
+# TF AWS Provider Core Services 
```

Comment:
Since this is not an official GitHub file, this should live in a new `docs/` directory. 👍